src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+
+    private static final Random random = new Random(47);
+
     public String getRandomColor() {
-        return null;
+        Color[] colors = Color.values();
+        return colors[random.nextInt(colors.length)].toString();
```

Comment:
Use the name() method instead of toString() when you need a String representation of an enum. The name() method will always return the exact name as declared in the enum, which is the preferred behavior.